handbook/onboarding/leads.md

```diff
@@ -17,6 +17,14 @@ Everyone should join a **private** channel named `#tm-leads-YYYY` (for example,
 
 As a lead, you should also lead in Slack participation by example - please make sure you join any channels you find interesting and participate in them! See [our Slack guide](/handbook/tools/slack.md#finding-channels) for more details.
 
+### Slack Status
+
+To help people identify you, make sure you set a status in Slack indicating your role!
+
+![update status](img/update-status.png)
```

Comment:
Am I supposed to be able to see these images in the Netlify preview? They're not showing up.

Reply:
Nice catch, thanks!! Looks like vuepress requires relative paths here - fixed in bb42ea1c0753bfaadb52ebbad77534db9e2b5eca